packages/runtime/src/tutorial-runner.ts

```diff
@@ -314,26 +314,32 @@ export class TutorialRunner {
     // first add template files
     for (const [filePath, value] of Object.entries(this._currentTemplate || {})) {
       if (typeof value === 'string') {
-        files[removeLeadingSlash(filePath)] = value;
+        files[filePath.slice(1)] = value;
       }
     }
 
     // next overwrite with files from editor
     for (const [filePath, value] of Object.entries(this._currentFiles || {})) {
       if (typeof value === 'string') {
-        files[removeLeadingSlash(filePath)] = value;
+        files[filePath.slice(1)] = value;
       }
     }
 
-    const packageJson = parseJson(files['package.json']);
-
-    // add start commands
     if (files['package.json']) {
-      const mainCommand = this._currentRunCommands?.mainCommand?.shellCommand;
-      const prepareCommands = (this._currentRunCommands?.prepareCommands || []).map((c) => c.shellCommand);
-      const startCommand = [...prepareCommands, mainCommand].filter(Boolean).join(' && ');
+      let packageJson;
+
+      try {
+        packageJson = JSON.parse(files['package.json']);
+      } catch {}
+
+      // add start commands when missing
+      if (packageJson && !packageJson.stackblitz?.startCommand) {
```

Comment:
```suggestion
      if (packageJson && !packageJson.scripts?.start && !packageJson.scripts?.dev) {
```

Reply:
Sure! Though I wonder if tutorial's `prepareCommands` could have something that project's `start` and `dev` scripts don't cover. 🤔

Reply:
Yeah that's a good point, let's keep it like you have done. We can always revisit that part later.